frontend/src/metabase-lib/metadata.ts

```diff
@@ -153,9 +153,17 @@ declare function DisplayInfoFn(
   query: Query,
 ): QueryDisplayInfo;
 
+declare function DisplayInfoFn(query: Query): QueryDisplayInfo;
+
 // x can be any sort of opaque object, e.g. a clause or metadata map. Values returned depend on what you pass in, but it
 // should always have display_name... see :metabase.lib.metadata.calculation/display-info schema
-export const displayInfo: typeof DisplayInfoFn = ML.display_info;
+export const displayInfo: typeof DisplayInfoFn = (...args: unknown[]) => {
+  if (args.length === 1) {
+    return ML.display_info(undefined, undefined, ...args);
```

Comment:
I think you'll need to pass query and stageIndex anyway

```
const query = args[0]
const stageIndex = -1;

return ML.display_info(query, stageIndex, query)
```

Reply:
That was the most surprising thing - you don't 😮 
![image](https://github.com/metabase/metabase/assets/31325167/ce75be9b-2b78-48e6-9080-505bccfb9f2d)

Reply:
@snoe confirmed in Slack that this currently works "only because Malli validation is not turned on in cljs".

So this is a dead end attempt.
We should forget about another function overload for now. Either we create a named abstraction (e.g. `queryDisplayInfo`) or we wait to see if these named, simplified functions start appearing in backend.

cc @kamilmielnik @uladzimirdev

Reply:
`queryDisplayInfo` :+1:

Reply:
Split this step into a separate PR:
https://github.com/metabase/metabase/pull/37862